setup/providers/kubernetes-v2/index.md

```diff
@@ -59,19 +59,13 @@ metadata:
  name: spinnaker-role
 rules:
 - apiGroups: [""]
-  resources: ["namespaces", "configmaps", "events", "replicationcontrollers", "serviceaccounts", "pods/logs"]
```

Comment:
Events, replicationcontrollers, serviceaccounts, and pods/logs only seem to be used by v1.

Reply:
I'm pretty sure `pods/logs` is used by V2 as well: 

https://github.com/spinnaker/clouddriver/blob/1376edebf7f6776202e6ec15925e0ea280274b65/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesV2InstanceProvider.java#L121-L133

Reply:
Thanks. Added.